README.md

```diff
@@ -13,6 +13,8 @@ subject to change, and some features are not fully implemented.
 
 ## Requirements
 
+The host or VM running Preflight must have RHEL 8.5 or equivalent (e.g. Fedora 35) installed.
```

Comment:
I'd be ok with mentioning a CentOS version here as well. But, I would also not use the "or equivalent" language. Just state the minimum version.

Reply:
Added